### PR TITLE
[Redesign] Add no-tickets message

### LINKED ISSUE #1307: [Redesign] Add "No ... tickets" text to the pages with no tickets to show
Add "No ... tickets" text to the pages with no tickets to show
image

Example (16px, Regular, #596D81): 
image

_Originally posted by @MariaPleshkova in https://github.com/decred/politeiagui/pull/1304#issuecomment-512272256_

### PR DESCRIPTION
**Pull request by victorgcramos**
Closes #1307

### CODE REVIEW

File: src/componentsv2/HelpMessage/HelpMessage.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Text, classNames } from "pi-ui";
+import React from "react";
+// import { NavLink } from "react-router-dom";
```
Comment (tiagoalvesdulce) [inline src/componentsv2/HelpMessage/HelpMessage.jsx]: This can be erased

File: src/containers/User/Detail/Proposals/Proposals.jsx [hunk 2]
```diff
@@ -11,7 +12,9 @@ const Proposals = props => {
   // TODO: need a loading while user has not been fetched yet
   return (
     !!proposals &&
-    !loading && <>{proposals.map(proposal => renderProposal(proposal))}</>
+    !loading && proposals.length > 0 ?
```
Comment (tiagoalvesdulce) [inline src/containers/User/Detail/Proposals/Proposals.jsx]: I'd say we should create a variable for this like: `const proposalsEmpty = !!proposals && !loading && proposals.length === 0` and use it like: `proposalsEmpty ? <HelpMessage>You have not created any proposals yet</HelpMessage> : {proposals.map(proposal => renderProposal(proposal))}`. We should also verify if it's loading before showing the HelpMessage or the proposals to properly show a loader.

File: src/containers/User/Detail/Proposals/Proposals.jsx [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
 
-  // TODO: need a loading while user has not been fetched yet
-  return (
-    !!proposals &&
-    !loading && <>{proposals.map(proposal => renderProposal(proposal))}</>
-  );
+  const emptyProposalList = !!proposals && proposals.length > 0;
+  return !loading ? emptyProposalList ?
+      <>{proposals.map(proposal => renderProposal(proposal))}</> :
+      <HelpMessage>You have not created any proposals yet</HelpMessage> :
```
[the remark this answers is not in the document]
Comment (victorgcramos) [reply src/containers/User/Detail/Proposals/Proposals.jsx]: Good observation, @fernandoabolafio! I'll address this! Thank you!

**Comment by victorgcramos** [comment]
What about removing the "Fetching Proposal" text? I think it's redundant since it's clearly fetching use proposals...

**Comment by fernandoabolafio** [comment]
@victorgcramos please, fix the conflicts before we move this in.

**Pull request merged**